scripts/Muon/GUI/Common/fitting_tab_widget/fitting_tab_view.py

```diff
@@ -94,29 +161,22 @@ def setup_fit_options_table(self):
             ("Property;Value").split(";"))
 
         table_utils.setRowName(self.fit_options_table, 0, "Time Start")
-        self.minimizer_combo = table_utils.addDoubleToTable(self.fit_options_table, 0.0, 0, 1)
+        self.time_start = table_utils.addDoubleToTable(self.fit_options_table, 0.0, 0, 1)
 
         table_utils.setRowName(self.fit_options_table, 1, "Time End")
-        self.minimizer_combo = table_utils.addDoubleToTable(self.fit_options_table, 15.0, 1, 1)
+        self.time_end = table_utils.addDoubleToTable(self.fit_options_table, 15.0, 1, 1)
 
         table_utils.setRowName(self.fit_options_table, 2, "Minimizer")
         self.minimizer_combo = table_utils.addComboToTable(self.fit_options_table, 2, [])
 
-        # table_utils.setRowName(self.fit_options_table, 3, "TF Asymmetry Mode")
-        # self.tf_asymmetry_mode_checkbox = table_utils.addCheckBoxWidgetToTable(
-        #     self.fit_options_table, False, 3)
-        #
-        # table_utils.setRowName(self.fit_options_table, 4, "Plot Difference")
-        # self.plot_differences_checkbox = table_utils.addCheckBoxWidgetToTable(
-        #     self.fit_options_table, True, 4)
+        self.minimizer_combo.addItems(
+            ['Levenberg-Marquardt', 'BFGS', 'Conjugate gradient (Fletcher-Reeves imp.)', 'Conjugate gradient (Polak-Ribiere imp.)',
+             'Damped GaussNewton', 'FABADA', 'Levenberg-MarquardtMD', 'Simplex',
+             'SteepestDescent', 'Trust Region'])
```

Comment:
lets put this list in either its own file or at the top of this one. Then it is easy to find and update